Cargo.toml

```diff
@@ -55,6 +55,8 @@ parley = { version = "0.6.0", features = ["accesskit"] }
 peniko = "0.5.0"
 winit = "0.30.12"
 tracing = { version = "0.1.41", default-features = false }
+tracing-wasm = "0.2.1"
+console_error_panic_hook = "0.1.2"
```

Comment:
Why this version? It's not what's in Cargo.lock, which we generally want to synchronise our direct dependencies to.